rest-servlet/src/main/java/org/hawkular/inventory/rest/RestResourceTypes.java

```diff
@@ -112,8 +113,9 @@ public ResourceType get(@PathParam("tenantId") String tenantId,
     public Set<Resource> getResources(@PathParam("tenantId") String tenantId,
                                       @PathParam("resourceTypeId") String resourceTypeId) {
 
-        return inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId)
-                .resources().getAll().entities();
+        ResourceTypes.Single single = inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId);
```

Comment:
k, you persuaded me :smile:  But in that case we should do the same thing for all other entity types...

http://.../inventory/no-tenant/environments should also return 404, etc.